.ipynb_checkpoints/gensim Quick Start-checkpoint.ipynb

```diff
@@ -0,0 +1,389 @@
+{
```

Comment:
I do not understand what you mean by this.
I searched my repo for ".ipynb_checkpoints/*" but couldn't find a single match.
Perhaps you are looking at an old commit.

I opened the ipython notebook initally on my computer which might have caused these changes.

Is there any way to edit notebooks without affecting default settings?